src/modules/infobox/components/ImportContent.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+    <div data-infobox="height-reference" class="import-overlay-content">
+        <div class="input-group d-flex">
+            <input
+                type="text"
+                class="form-control search-bar-input"
+                :placeholder="$t('import_online_placeholder')"
+                :value="importValue"
+                data-cy="import"
+                @input="onInputChange"
+                @focus="showProviders"
+                @keydown.down.prevent="goToFirstResult"
+                @keydown.esc.prevent="hideProviders"
+            />
```

Comment:
On the input it would be good to have a clear button inside the input to clear it

![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/d5f993cb-6f0d-4ebf-9a28-57b933957302)

Similar to the clear button of the search bar 
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/33ba5fa9-1dda-4ab7-bb1f-83b4f161b5eb)

The clear button should only appear if there is anything to clear.

Reply:
Would be also nice to have a toggle button for the provider list as on the old viewer.

Reply:
> On the input it would be good to have a clear button inside the input to clear it
It will be better to create an issue for this and describe how the button should behave when a layer is selected and parsed

> Would be also nice to have a toggle button for the provider list as on the old viewer.
according to the issue description, we shouldn't implement this switch

Reply:
> > Would be also nice to have a toggle button for the provider list as on the old viewer.

> according to the issue description, we shouldn't implement this switch

I was thinking of a dropdown button in the input bar in order to open/close the list of provider not the tab for `online` and `local`

Reply:
Ok about the clear button, you are right this needs some thought and refinement.

Reply:
> I was thinking of a dropdown button in the input bar in order to open/close the list of provider not the tab for online and local

Ah ok, I added a toggle button